### PR TITLE
Changes for production Jenkins

### PR DESCRIPTION
**Pull request by MancunianSam**
- Policy changes to get prod jenkins working
- Set the private ip when the variable is there.
- Add permissions for the prod cluster
- Remove prod and staging roles from the integration fargate role

### CODE REVIEW

File: ecs/variables.tf [hunk 1]
```diff
@@ -81,6 +85,10 @@ variable "task_role_arn" {
   default = ""
 }
 
+variable "name" {
```
Comment (TomJKing) [inline ecs/variables.tf:88]: Might be worth making this a little more descriptive and/or adding a description

File: iam_policy/templates/jenkins_backup_prod_s3_policy.json.tpl [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (TomJKing) [inline iam_policy/templates/jenkins_backup_prod_s3_policy.json.tpl:1]: Maybe order alphabetically.

File: iam_policy/templates/jenkins_ecs_task_integration.json.tpl [hunk 3]
```diff
@@ -8,6 +8,8 @@
       "Resource": [
         "arn:aws:s3:::tdr-jenkins-backup-mgmt",
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/mgmt/access_key",
+        "arn:aws:ssm:eu-west-2:${account_id}:parameter/mgmt/github/prod/secret",
```
Comment (TomJKing) [inline iam_policy/templates/jenkins_ecs_task_integration.json.tpl:11]: Maybe order alphabetically

File: iam_policy/templates/jenkins_fargate_prod.json.tpl [hunk 4]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (TomJKing) [inline iam_policy/templates/jenkins_fargate_prod.json.tpl:1]: Maybe order alphabetically.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just some minor comments

File: ecs/variables.tf (hunk 1 above)
Comment (MancunianSam) [reply ecs/variables.tf:88]: I've added a description in.

File: iam_policy/templates/jenkins_ecs_execution_prod.json.tpl [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (TomJKing) [inline iam_policy/templates/jenkins_ecs_execution_prod.json.tpl:1]: Maybe order alphabetically.

**Review by TomJKing — COMMENTED** [review]
Something else I noticed. Very minor

File: iam_policy/templates/jenkins_ecs_execution_prod.json.tpl (hunk 5 above)
Comment (MancunianSam) [reply iam_policy/templates/jenkins_ecs_execution_prod.json.tpl:1]: I've sorted the resources into order.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**